Run-SystemTest.ps1

```diff
@@ -36,7 +36,9 @@ Process {
     $outputDir = Join-Path $PSScriptRoot ".output"
     
     $contractFirstSysTestDir = Join-Path $PSScriptRoot "src/tests/WebApi.MsBuild.SystemTest.ContractFirst"
+    $validateContractSysTestDir = Join-Path $PSScriptRoot "src/tests/WebApi.MsBuild.SystemTest.ValidateContract"
```

Comment:
Could be interesting to stop the spread of system test project to leverage the extra args property to be able to re-use the same project

By exemple on project CodeFirst:
- First call with Mode: CodeFirst
- Then call with Mode: Generate

Reply:
Out of scope for this PR. Should be done in a seperate issue.